SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1170,7 +1170,7 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 
 	// True if the entity is in the worker's view.
 	// If this is the case then we know the entity was created and do not need to retry if the request timed-out.
-	const bool bEntityIsInView = NetDriver->StaticComponentView->HasComponent(SpatialGDK::Position::ComponentId, GetEntityId());
+	const bool bEntityIsInView = NetDriver->Connection->GetCoordinator().HasComponent(SpatialGDK::Position::ComponentId, GetEntityId());
```

Comment:
how do you feel about this asking the actorsystem instead- this might return true when we don't expect it, e.g. entity is checked out by another system that the actor system

Reply:
Would be cool to do this by using a subview/ActorSystem 100%; would like to make this PR focused on getting rid of StaticComponentView if I can.